### PR TITLE
add: made PriceType optional and defaults to fixed

### PR DESCRIPTION
**Pull request by diyar214**

### CODE REVIEW

**Review by MohammedLilo — COMMENTED** [review]
Marvelous!, 

this amount of love is sus ! / yousif

**Review by Yousif-Alsheikh — APPROVED** [review]

File: Migrations/MainAppContextModelSnapshot.cs
Comment (KAJOiq) [inline Migrations/MainAppContextModelSnapshot.cs:1]: Great job!

File: Models/DTOs/ProjectInputDTO.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public class ProjectInputDto
 
         [Required]
```
Comment (KAJOiq) [inline Models/DTOs/ProjectInputDTO.cs:26]: NIGGA WHAT ?????????????????????????

File: Models/DTOs/ProjectInputDTO.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public class ProjectInputDto
...
         [AllowedValues(Constants.PROJECT_PRICETYPE_PERHOUR, Constants.PROJECT_PRICETYPE_FIXED, ErrorMessage ="Price type is invalid.")]
-        public string PriceType { get; set; }
+        public string? PriceType { get; set; }
```
Comment (KAJOiq) [inline Models/DTOs/ProjectInputDTO.cs:28]: You made it optional NIGGAAAAAAAAAA!

**Review by KAJOiq — COMMENTED** [review]
You need to resolve your skill issue :) NIGGAAAAAAAA

**Comment by KAJOiq** [comment]
Please resolve conflict and then push your code again.

File: Models/DTOs/ProjectInputDTO.cs (hunk 2 above)
Comment (MohammedLilo) [reply Models/DTOs/ProjectInputDTO.cs:28]: Yes, because this was part of his task, the price type is no longer required and it is `fixed` by default

Comment (KAJOiq) [reply Models/DTOs/ProjectInputDTO.cs:28]: Yeah i know, But it is still used `[Required]`.
It should be made Constant and without using [Required]. 
If it is `fixed` by default, why is it required?

**Comment by KAJOiq** [comment]
> #### Description
> Need to do the following in this PR:
> 
> * [x]  **1.** Make the price type optional and default to `fixed`.
> * [ ]  **2.** Implement a profile picture for the user. Use a `PATCH` request for uploading the image. Include the image URL (if it exists) in the user profile and any relevant place.

File: Models/DTOs/ProjectInputDTO.cs (hunk 2 above)
Comment (MohammedLilo) [reply Models/DTOs/ProjectInputDTO.cs:28]: Yeah, he should have deleted `Required`, although it doesn't affect the wanted outcome, it is still a skill issue.
But wdy by making it "Constant"?

Comment (KAJOiq) [reply Models/DTOs/ProjectInputDTO.cs:28]: It is still a skill issue 😂. 

He can make it easly by using `Constants.PROJECT_PRICETYPE_FIXED`, i guess we have it in `Constants` class.
But as a default PriceType is `fixed` right now so we don't have this change.

**Pull request closed without merging**